2-RTB/Verbali/Interni/Verbale_2024_01_02_v1.pdf/resoconto.tex

```diff
@@ -0,0 +1,38 @@
+\section{Resoconto} \label{sec:resoconto}
+\subsection{Discussioni} \label{subsec:resdiscussione}
+\begin{enumerate}
+    \item La riunione è iniziata con la fase di retrospettiva, nella quale il gruppo ha esaminato il precedente sprint, valutando il raggiungimento degli obiettivi prefissati e valutando la documentazione attuale.
+    Da questa prima sezione è emerso che i vari obiettivi prefissati sono stati raggiunti e che la documentazione abbia raggiunto un ottimo stato di maturità.  Si è inoltre discusso sullo stato di avanzamento del \ccgloss{PoC} che ha raggiunto tutti gli obiettivi prefissati per lo sprint precedente.
+    \item Si è successivamente discusso, con un recap, lo stato dei documenti da presentare nell'\ccgloss{RTB}. Si è posta particolare attenzione alle sezioni mancanti dei vari documenti e sui documenti Piano\_di\_qualifica e Norme\_di\_progetto. 
+    Infine tramite una discussione interna si è deciso di chiedere al professore Cardin, \ccgloss{committente} del progetto, un incontro per presentare e discutere le modifiche apportate all'Analisi\_dei\_requisiti in base ai suggerimenti dati durante il colloquio precedente.
+    \item In una seconda fase della riunione si è visionato il \ccgloss{PoC}, valutandone lo stato attuale e le funzionalità contenute in esso. Nonostante lo stato finale in cui si trova si e' fatto luce su alcune mancanze che saranno obiettivi dello sprint corrente.
```

Comment:
In una seconda fase della riunione si è visionato il \ccgloss{PoC}, valutandone lo stato attuale e le funzionalità contenute in esso. Nonostante lo stato molto avanzato in cui si trova, si è fatto luce su alcune mancanze che saranno obiettivi dello sprint corrente.